app/src/main/java/com/github/se/eventradar/ui/hosting/CreateEvent.kt

```diff
@@ -0,0 +1,299 @@
+                  modifier =
+                      Modifier.fillMaxWidth()
+                          .padding(start = 35.dp, top = 8.dp, end = 35.dp)
+                          .testTag("organisersMultiDropDownMenuTextField"),
+                  getFriends = viewModel::getHostFriendList,
+                  friendsList = uiState.hostFriendsList)
+              // add a Button!
+              // successful or not?
+              Button(
+                  onClick = {
+                    if (viewModel.validateFields()) {
+                      viewModel.addEvent()
+                    }
+                  },
+                  modifier =
+                      Modifier.wrapContentSize()
+                          .fillMaxWidth()
+                          .padding(start = 35.dp, top = 16.dp, end = 35.dp)
+                          .testTag("publishEventButton"),
+                  border = BorderStroke(width = 1.dp, color = MaterialTheme.colorScheme.primary),
+                  colors =
+                      ButtonDefaults.buttonColors(
+                          containerColor = MaterialTheme.colorScheme.primary),
+              ) {
+                Text(
+                    text = "Publish Event",
+                    style =
+                        TextStyle(
+                            fontSize = 14.sp,
+                            lineHeight = 17.sp,
+                            fontFamily = FontFamily.SansSerif,
+                            fontWeight = FontWeight(500),
+                            color = Color(0xFFFFFFFF),
+                            letterSpacing = 0.25.sp,
+                        ),
+                    modifier = Modifier.padding(start = 8.dp),
+                )
+              }
+              StandardDialogBox(
+                  display = (uiState.showAddEventSuccess),
+                  modifier = Modifier.testTag("successDialogBox"),
+                  title = "Successfully Created Event",
+                  message = "You have succesfully created your event, invite others over to join!",
+                  onClickConfirmButton = {
+                    // update the error states!
+                    viewModel.resetStateAndSetAddEventSuccess(false)
+                    navigationActions.goBack()
+                  },
+                  boxIcon = { Icon(Icons.Default.Check, "Success Icon") })
+              StandardDialogBox(
+                  display = (uiState.showAddEventFailure),
+                  modifier = Modifier.testTag("failureDialogBox"),
+                  title = "Creating Event has Failed",
+                  message = "Something went wrong with creating your event, please retry.",
+                  onClickConfirmButton = {
+                    viewModel.resetStateAndSetAddEventFailure(false)
+                    navigationActions.goBack()
+                  }, // reset or not idk!
+                  boxIcon = { Icon(Icons.Default.Clear, "Failure Icon") })
+            }
+      }
+}
+
```

Comment:
This was included in the covrage, you can add: "@ExcludeFromJacocoGeneratedReport" above such that it is not.